tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -56,6 +56,7 @@ public Security(SecuritySettings? settings = null, IWaf? waf = null, IDictionary
             if (actions != null)
             {
                 _configurationStatus.Actions = actions;
+                _configurationStatus.Actions = actions;
```

Comment:
Just to be extra sure:
```suggestion
                _configurationStatus.Actions = actions;
                _configurationStatus.Actions = actions;
                _configurationStatus.Actions = actions;
                _configurationStatus.Actions = actions;
                _configurationStatus.Actions = actions;
                _configurationStatus.Actions = actions;
```

Reply:
😝